Kernel/include/scheduler.h

```diff
@@ -0,0 +1,11 @@
+#include <stdint.h>
+
+void sch_init();
+uint64_t sch_switchProcess(uint64_t * currentRSP);
+int sch_loadProcess(uint64_t functionAddress, uint16_t priority,int argq, char * args[]);
```

Comment:
```suggestion
int sch_loadProcess(void* functionAddress, uint16_t priority, int argc, char* argv[]);
```
Some things to think about:
- Should argv be a `const char*[]` instead?
- Where are those strings stored? Who's in charge of their lifecycle?